Pidaq/README.md

```diff
@@ -1,8 +1,20 @@
 # Pidaq (Rasp Pi Data Aquisitor) 
 
 This directory contains all of the code that will run on the raspberry pi. The pi will receive and parse CAN messages containing raw telemetry, apply calculations,  and create a packet(s) containing either a JSON or protobuf message to send to the control laptop via ethernet (UDP)
-*** 
-## Examples
+ 
+# Development
+We will be using docker to cross compile c++ for the raspberry pi
 
-Includes relevant examples 
-***
+Ensure docker is installed and running on your machine. Windows 10 Home does not have proper support for docker, however MUN offers a free upgrade to windows 10 education which is fully featured. The update is easy, unintrusive, and tasks less than 1 hour.
```

Comment:
Lil typo here, "tasks" should be "takes"

Reply:
Fixed